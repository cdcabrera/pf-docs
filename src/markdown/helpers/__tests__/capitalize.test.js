const { capitalize } = require('../capitalize');

describe('Capitalize', () => {
  it('should capitalize', () => {
    expect(capitalize('lorem ipsum dolor sit')).toMatchSnapshot('capitalize');
  });
});
