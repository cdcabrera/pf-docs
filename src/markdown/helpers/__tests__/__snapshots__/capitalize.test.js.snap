// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Capitalize should capitalize: capitalize 1`] = `"Lorem ipsum dolor sit"`;
